reference/Upload-API.yaml

```diff
@@ -879,5 +880,352 @@ paths:
                       etag: 55a64df551425fcc55e4d42a148795d9f25f89d6
         description: The request body should contain a reference to each part along with the Etag of each upload.
     parameters: []
+  '/api/v1/{apiUserId}/{apiKey}/{targetId}/{languageCode}/search':
+    parameters:
+      - schema:
+          type: string
+        name: apiUserId
+        in: path
+        required: true
+        description: Guid identifying the requesting client
+      - schema:
+          type: string
+        name: apiKey
+        in: path
+        required: true
+        description: Second layer encrypted identifier for the requesting client
+      - schema:
+          type: string
+        name: targetId
+        in: path
+        description: Guid identifying the target where articles are allowed to be published
+        required: true
+      - schema:
+          type: string
+        name: languageCode
+        in: path
+        required: true
+        description: Two characters ISO language code
+    get:
+      summary: Article search
+      tags:
+        - search
```

Comment:
On top it's `Search` (uppercase 'S'), here it is `search` (lowercase 's')

can we please make it uppercase here?

Reply:
Yes sure